### ISSUE TITLE
Vektor Skin "Classic": Bugs

### ISSUE THREAD

**Issue by cyanit** [kurasu/surge#221]
Since #23 is a bit too long, please post bugs you encounter with the "Classic" skin here.

**Currently on my to-do list:**

- check the save dialog size
- more subtle red (mute)
- better visible orange (solo)
- make higher contrast sliders
- maybe bring back the “+“ slider endings
- adjust the faders triangle fills
- brighten up the “off-gradient“

**Comment by cyanit** [comment]
**Store Patch** (Done)
Affinity Designer Imported the bitmap at the wrong size. tried again, still does. If anything else looks wrong let me know, it might be the same issue.

**Comment by cyanit** [comment]
okay, great.

**Comment by itsmedavep** [comment]
As mentioned on #23 

Ah. Maybe we don't want to work this way, but I was thinking there should be individual bug/improvement reports. That way ANY designer can hop in and pick things off. When its put all under a single issue its a bit more difficult to manage collaboration.

@cyanit Has now created a bug/improvement list in a single issue. 🤷‍♂️ We now have to parse a bug/feature list to find an issue then inquire about the state items on it to take on items to fix? 

Seem much less collaborative. ;-[ Or is the intent to only have a single person do this?

**Comment by cyanit** [comment]
@itsmedavep uh, right. Didn't Thought that through... Just was the quickest way (for me) to keep track of what's broken, what needs a fix and what's next. Ofc this kinda breaks collaboration. Ill update the issue.

**Comment by baconpaul** [comment]
Oh hey I figured out the envelope thing.

So basically if you switch to "analog" all the envelope images (image 145) turn off displaying what's on the background.

The background image in resources/bitmaps/ 102 looks like this

<img width="193" alt="screen shot 2019-01-05 at 7 03 38 pm" src="https://user-images.githubusercontent.com/13387561/50730491-9e0e3b00-111c-11e9-9f39-5908834588bf.png">

the same thing in assets/classic-skin/export looks like this

<img width="187" alt="screen shot 2019-01-05 at 7 04 00 pm" src="https://user-images.githubusercontent.com/13387561/50730499-acf4ed80-111c-11e9-90a1-6eed2d77310c.png">

so we just need to draw that ADSR envelope in that spot for both amp and filter in the background and then it will work. Or "work". Bit of a hack tbh but it's the hack we have!

**Comment by baconpaul** [comment]
<img width="388" alt="screen shot 2019-01-05 at 7 06 09 pm" src="https://user-images.githubusercontent.com/13387561/50730508-f7766a00-111c-11e9-8c12-bfdeaab7b1d2.png">

this background asset is the reason the lfo type image looks different than the rest of the controls

<img width="208" alt="screen shot 2019-01-05 at 7 06 35 pm" src="https://user-images.githubusercontent.com/13387561/50730514-06f5b300-111d-11e9-9c78-3c755aa6b794.png">

I think that background needs to be updated to match the look and feel of other controls in the background.

**Comment by cyanit** [comment]
BTW: Is there another way to update my fork or do I need to delete -> re-fork -> request all the time?

**Comment by baconpaul** [comment]
Oh sure

Never develop on master and then

Git remote add upstream https://GitHub.com/Kira’s/surge 

Then when you want to update do


Git fetch upstream

Git checkout master

Git merge upstream/master

Git push origin master

On my phone - pls correct typos with the obvious corrrect stuff

**Comment by cyanit** [comment]
New to git and all its stuff, thanks for the help. Lets see if Github Desktop can do this too...

One more thing: could I make a script (“update repo“) that runs these commands when you run it? Or does git require a special command line (i heard about git bash)

**Comment by baconpaul** [comment]
No; just some of them are sticky so you don't need to do them. You only need to add the remote once.

```
git fetch upstream
git checkout master
git merge upstream/master
git push origin
```

is something I do type a lot.

There's a git --track thing which may do this more automatically but I never looked at that.

**Comment by baconpaul** [comment]
Did the PR.

It looks really nice.

<img width="910" alt="screen shot 2019-01-05 at 8 01 44 pm" src="https://user-images.githubusercontent.com/13387561/50730833-bbdf9e00-1124-11e9-8c8f-9994b666df20.png">

Still think that red on mute is a bit to red (and the white on unmuted is a bit too white). But already looks sharper.

**Comment by esaruoho** [comment]
@itsmedavep  yeah, definitely "asym" would be better than "assym" - if someone can knock that in, that'd be great :)

**Comment by esaruoho** [comment]
@itsmedavep but about closing #182 - it's best to close a ticket when it has been implemented - unless if it turned into an endless conversation and it's time to reboot the conversation.
this way we know when some new feature has been fixed/merged in, like "assym->asym" :)

**Comment by esaruoho** [comment]
pls let's make keytrack/retrigger/ -3 -2 -1 0 +1 +2 +3 w-table orange again (left is new, right is old)
<img width="385" alt="vember_audio__surge_and_50730833-bbdf9e00-1124-11e9-8c8f-9994b666df20_png_1 820x1 134_pixels" src="https://user-images.githubusercontent.com/4966687/50733912-b9218f00-119e-11e9-8088-6f864ca5d5ea.png">

**Comment by cyanit** [comment]
Most stuff is intend to increase contrast, the white sliders on a dark gb for example.

The green was just not fitting (and the difference to subtle) in a Orange/Blue/Grey theme, so I exchanged it for Yellow/Red and made the inbetween state orange, which (in my opinion) is also very intuitive.

With bad eyesight, you don't want to read white text on a bright orange. Its horrible.

The “split lines“ are also not there, because I thought that every user has common sense. (Orange part = 1/3, two more words -> two other options)


The leaky white is supposed to be a triangle. Looked good on my mock. Guess ill remove it.
The sublte underline will be fixed, must have forgotten that yesterday.
Also, I agree that the sliders are still not dark enough. 
The pitch notches are something that is a bit frustraing, will solve that too. 

However, not today. The soonest fix will be within a few days, I have to get my own project done first.

**Comment by esaruoho** [comment]
@funkaloss the copypaste is from a previous ticket where the discussion was "change text, or change image", which @baconpaul clarified as "change image" since the `ASSYM` is not a string in code that can be changed :) the copypaste from previous ticket is just to make sure that the change is done and not forgotten

**Comment by funkaloss** [comment]
@esaruoho Oh, I understand. Thank you!

**Comment by esaruoho** [comment]
can this ticket be closed, @cyanit ? what do you think?

**Comment by cyanit** [comment]
@itsmedavep yours is working and done, so I would say yes. Right?
